app/views/capture/index2.html.erb

```diff
@@ -218,6 +218,7 @@
                   <a rel="tooltip" title="Default mode" class="btn btn-default btn-sm" onClick="$W.toggle_mode()"><i class="fa fa-circle-o"></i></a>
                   <a rel="tooltip" title="RGB mode" class="btn btn-default btn-sm" onClick="$W.toggle_mode()"><i class="fa fa-adjust"></i></a>
                   <a rel="tooltip" title="Combined readout" class="btn btn-default btn-sm" onClick="$W.show_combined()"><i class="fa fa-list-ul"></i></a>
+                  <a rel="tooltip" title="Detected fluorescent light" class="btn btn-default btn-sm" onClick=""><i class="fa fa-lightbulb-o"></i></a>
```

Comment:
Hi @jywarren @Tlazypanda @TildaDares 
I added this icon to detect fluorescent light on the v2 interface. It is present on the old capture interface but with no functionality. The onClick event listener is empty.

I was thinking that in order to get this to work, we have to add a function to detect fluorescent light into the spectral_workbench.js library before bringing it into the rails apps.

I wish to find out if this function exists already or if not how do I approach writing a function like this?
Was thinking we might have to compare the wavelength of the spectrum to maybe the standard wavelength of LEDs...?

Thank you

Reply:
Hi @jywarren, 
Adding `id="cfl-detect"` to the button causes the button to turn red anytime the spectrum detected is a fluorescent light. Do you think this fixes the issue?

Reply:
Yes, that's the correct behavior! Thank you @Georjane - i think this is fixed then!